db/warehouse/migrate/20240221195839_migrate_auto_exit_rules.rb

```diff
@@ -0,0 +1,36 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+class MigrateAutoExitRules < ActiveRecord::Migration[6.1]
+  def up
+    Hmis::AutoExitConfig.all.each do |old_auto_exit_config|
```

Comment:
Lets do this in a transaction, so that it rolls back if any of the operations fail:
```
  Hmis::AutoExitConfig.transaction do

   # 1) iterate and create
 
   # 2) destroy
   Hmis::AutoExitConfig.each(&:destroy!)
  end
```